src/Code.gs

```diff
@@ -368,6 +368,7 @@ FirebaseApp_._methodWhiteList = {
 FirebaseApp_._ERROR_TRY_AGAIN = "We're sorry, a server error occurred. Please wait a bit and try again.";
 FirebaseApp_._ERROR_GLOBAL_CRASH = "We're sorry, a server error occurred. Please wait a bit and try again.";
 FirebaseApp_._PERMISSION_DENIED = "Permission denied";
+FirebaseApp_._INVALID_DATA = "Invalid data; couldn't parse JSON object. Are you sending a JSON object with valid key names?";
```

Comment:
How should we handle this error? For me it mostly means that we should not perform a retry. Maybe better to copy what we have done in the ErrorHandler lib? (where we indicates if a retry is needed on each error type / message)

Reply:
Yes, no retry should be done, as the situation will not change.
Using the ErrorHandler lib pattern is appropriate here indeed (it will be really clear)